compiler/passes/codegen.cpp

```diff
@@ -532,6 +532,14 @@ static void codegen_header_compilation_config() {
 
 static void protectNameFromC(Symbol* sym) {
   //
+  // Symbols that start with 'chpl_' are presumably compiler-generated
+  // and sufficiently unique not to require further munging.
```

Comment:
I can't think of clean wording, but I think it should say compiler-generated or a symbol used by the internal modules